lib/smart_answer/calculators/child_maintenance_calculator.rb

```diff
@@ -34,11 +34,8 @@ def rate_type
     end
 
     def calculate_maintenance_payment
-      if @benefits == 'no'
-        send("calculate_#{rate_type}_rate_payment")
-      else
-        0 #irrelevant what we return, with benefits rate is either nil or flat
-      end
+      return 0 if state_benefits?
```

Comment:
Then can you at least include the comment? Otherwise it looks like `0` is an important value, when actually it isn't.